node/src/accountant/scanners/mod.rs

```diff
@@ -499,6 +497,19 @@ impl PayableScanner {
             panic!("{}", msg)
         };
     }
+
+    fn protect_payables(&self, payables: Vec<PayableAccount>) -> ProtectedPayables {
+        #[allow(clippy::unsound_collection_transmute)]
+        let bytes = unsafe { transmute::<Vec<PayableAccount>, Vec<u8>>(payables) };
+        ProtectedPayables(bytes)
+    }
+
+    fn expose_payables(&self, protected: ProtectedPayables) -> Vec<PayableAccount> {
+        #[allow(clippy::unsound_collection_transmute)]
+        unsafe {
+            transmute::<Vec<u8>, Vec<PayableAccount>>(protected.0)
+        }
+    }
```

Comment:
Add in another function pair, which will make up the inside of these two. They, for change, will be static function but with some kind of restricted visibility. Probably `pub (in crate::accountant)`. Then go to the right `test_utils` location and use it again in the test only method that is supposed to act a as helper in tests where we need to construct the protected payables but we shouldn't be able to in the production version.